apps/flame-defi/app/pool/types.ts

```diff
@@ -121,8 +119,8 @@ export type PoolPositionContextProps = {
   rawFeeTier: number; // The unformatted fee tier value to be used in calculations
   selectedSymbol: string; // The symbol of the token that is currently in the ToggleSwitch. Controls which set of token liquidity and price data is displayed
   handleReverseTokenData: (symbol: string) => void;
-  collectAsNative: boolean; // This boolean controls the toggle in the UI to collect fees as the native token
-  handleCollectAsNative: (collectAsNative: boolean) => void;
+  collectAsWrappedNative: boolean; // This boolean controls the toggle in the UI to collect fees as the native token
```

Comment:
```suggestion
  isCollectAsWrappedNative: boolean; // This boolean controls the toggle in the UI to collect fees as the native token
```
Name too confusing for a function. Prefix with `is` or `should`